src/widget.h

```diff
@@ -110,6 +112,23 @@ struct enum_traits<cardinal_direction> {
     static constexpr cardinal_direction last = cardinal_direction::num_cardinal_directions;
 };
 
+// Used when determining bodypart status indicators in sidebar widgets.
+enum class bodypart_status : int {
+    BITTEN = 0,
+    INFECTED = 1,
```

Comment:
It's probably fine to leave out the explicit numeric initializers for these. I doubt we're like to do anything funny like bit masks or anything.

Reply:
Oops, that's a leftover from when I was trying things out. Forgot to remove it apparently :P